src/main/java/uk/ac/ebi/atlas/download/FileDownloadController.java

```diff
@@ -144,4 +153,58 @@ public FileDownloadController(ExperimentFileLocationService experimentFileLocati
             zipOutputStream.close();
         }
     }
+
+    @RequestMapping(value = "json/experiments/check/zip",
```

Comment:
I think that in this case, a URL that follows the same pattern as above (i.e. `experiments/download/zip`) seems more appropriate. Maybe `experiments/download/zip/check` or `experiments/download/check`?

Reply:
I correct it as `experiments/download/zip/check`.